### DISCUSSION TITLE
Welcome to Mishos Discussions!

### DISCUSSION THREAD

**Discussion by bljohnsondev** [bljohnsondev/mishos#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Martin-Fox** [comment]
Hi,

I'm a simple guy, self-hosting what I can from my garage, and I'm mostly using other peoples work :)

And from time to time I'm adding some requests in some repos :)
